src/components/SchemaRow/TopLevelSchemaRow.tsx

```diff
@@ -23,6 +23,14 @@ export const TopLevelSchemaRow: React.FC<SchemaRowProps> = ({ schemaNode, nestin
   }
 
   if (isRegularNode(schemaNode) && choices.length > 1) {
+    let combinerType = '';
+    if (selectedChoice.type.subpath[0] === 'oneOf') {
```

Comment:
prob the same as above - could use `combiners` to determine it, but using `subpath` is good too.